test/types/NumberInput.tsx

```diff
@@ -43,7 +43,7 @@ function NumberInputOldWithAllProps() {
       noLeftBorderRadius
       noRightBorderRadius
       onBlur={_ev => {}}
-      onChange={_ev => {}}
+      onChange={(value: number) => {}}
```

Comment:
We need to see that typescript manages to infer the type of value + use `_` prefix so the linter won't complain about unused variable
```suggestion
      onChange={_value => {}}
```

Reply:
Hmm, I am not sure I agree with this - the test will not catch any issues if we leave untyped code like this. I could go to typings and change onChange argument type unintentionally and the tests would still say it's ok.